connectivity/manifests/client-egress-l7-http-method.yaml

```diff
@@ -0,0 +1,30 @@
+---
+# client2 is allowed to contact the echo Pod
+# on port 8080 via POST method. HTTP introspection is enabled for client2.
+# The toFQDNs section relies on DNS introspection being performed by
+# the client-egress-only-dns policy.
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  namespace: cilium-test
+  name: client-egress-l7-http-method
```

Comment:
nit: test name could be same as policy name for faster reference (e.g. client-egress-l7-method)